packages/0xcert-ethereum-gateway/src/lib/actions-order.ts

```
@@ -166,8 +169,10 @@ export function getActionProxy(gateway: Gateway, action: ActionsOrderAction) {
     return ActionsGatewayProxy.XCERT_CREATE;
   } else if (action.kind == ActionsOrderActionKind.SET_ABILITIES) {
     return ActionsGatewayProxy.MANAGE_ABILITIES;
-  } else {
+  } else if (action.kind == ActionsOrderActionKind.UPDATE_ASSET_IMPRINT) {
     return ActionsGatewayProxy.XCERT_UPDATE;
+  } else {
+    return ActionsGatewayProxy.XCERT_REVOKE;
   }
```

Please add `if` and `throw` on invalid `kind`.